packages/table/src/Row/Row.types.ts

```diff
@@ -20,9 +20,40 @@ export interface InternalRowWithRTProps<T extends LGRowData>
    */
   row: LeafyGreenTableRow<T>;
   /**
-   * Virtual row object passed from the `useLeafyGreenTable` hook
+   * Virtual row object passed from the `useLeafyGreenVirtualTable` hook
    */
   virtualRow?: VirtualItem;
+
+  /**
+   * Determines whether alternating rows will have dark backgrounds.
+   * @default false
+   */
+  shouldAlternateRowColor: boolean;
+
+  /**
+   * Whether the theme is dark or light
+   */
+  theme: Theme;
```

Comment:
This is related to what i mentioned above, memoizing does not work if i call context within the component so instead of calling `const { theme } = useDarkMode();` inside of `MemoizedInternalRowWithRT`, i pass it as a prop.

Reply:
I'm curious whether memoizing the components is still necessary after the other perf improvements you've made